tools/demo/demo_image.py

```diff
@@ -48,8 +48,18 @@ def postprocess_attr(dataset_attr_labelmap, label_list, conf_list):
     else:
         return [[], []]
 
+def restricted_float(x):
```

Comment:
can we find a better name, e.g., `annotate_image` for this script

Reply:
The name has been changed